CHANGELOG.md

```diff
@@ -17,6 +18,10 @@ A more detailed list of changes is available in the corresponding milestones for
   - **[com.google.fonts/check/superfamily/vertical_metrics]:** Included hhea.lineGap in comparison
   - **[com.google.fonts/check/glyph_coverage]:** Use glyphsets lib so we can improve this check in the future. (pull #3753)
   - **[com.google.fonts/check/fontbakery_version]:** If the request to PyPI.org is not successful (due to host errors, or lack of internet connection), the check fails. (pull #3756)
+  - **[com.google.fonts/check/varfont/regular_wght_coord]:** This check is no longer skipped when the value of "wght" axis coordinate of the "Regular" instance is not 400. (pull #3766)
+  - **[com.google.fonts/check/varfont/regular_wdth_coord]:** This check is no longer skipped when the value of "wdth" axis coordinate of the "Regular" instance is not 100. (pull #3766)
+  - **[com.google.fonts/check/varfont/regular_slnt_coord]:** This check is no longer skipped when the value of "slnt" axis coordinate of the "Regular" instance is not 0. (pull #3766)
+  - **[com.google.fonts/check/varfont/regular_ital_coord]:** This check is no longer skipped when the value of "ital" axis coordinate of the "Regular" instance is not 0. (pull #3766)
 
```

Comment:
These four descriptions on the CHANGELOG are incorrect.

Reply:
The checks are still skipped when a certain needed axis is missing. But now the SKIP is explicit in the check implementation, including a descriptive message of the reason why it was skipped. Before it would SKIP due to an unfulfilled condition without a more detailed explanation.

This is related to the feature proposal at #3277

Reply:
What's incorrect about them?

The checks included conditions (like [regular_wght_coord](https://github.com/googlefonts/fontbakery/pull/3766/files#diff-e46cbaca83ed6c15450bafcf9fd7498413272d046a1e9e0bb0b5bc0e5ea22419L26-L28)) which made them skip fonts that had the exact problems the check was meant to catch.
For example, [this FAIL](https://github.com/googlefonts/fontbakery/pull/3766/files#diff-e46cbaca83ed6c15450bafcf9fd7498413272d046a1e9e0bb0b5bc0e5ea22419L27-L28) could never be hit because of the `regular_wght_coord` condition; the check's result was a SKIP because of an unfulfilled condition.

Reply:
@felipesanches What's the action-item? Just change the copy? If so, to what?

Reply:
Here's the implementation of the `regular_wght_coord` condition:
![Screenshot from 2022-05-09 16-02-20](https://user-images.githubusercontent.com/213676/167479459-7d4082ad-5749-4d70-9d6e-79b124519109.png)

It looks up all fvar instances looking for one called "Regular". Then, it gets the value of its `wght` coordinate.

By "this FAIL" (reading the code test you linked in there) you're referring to the code-test that ensures the check detects when `'The "wght" axis coordinate of the "Regular" instance must be 400, but got 500 instead.'`

If the check was skipped due to an unfulfilled condition, it means the font lacks an instance named "Regular". It would probably be good to add an assert statement in the code-test to make sure we're using a "Regular" instance to test the check. Something like this:

```python
regular_instance = ttFont["fvar"].instances[0]
assert ttFont["name"].getDebugName(regular_instance.subfamilyNameID) == "Regular"
```

Reply:
> If the check was skipped due to an unfulfilled condition

It **used to be skipped** because `regular_wght_coord` was used as a condition for running the check. I removed that condition and now the check runs as expected, yielding FAIL (instead of SKIP) when it's supposed to.

> it means the font lacks an instance named "Regular"

It could mean that, yes. BUT it could also mean that the font lacks a `wght` axis, for example, because the `regular_wght_coord` will also return None in such case.

Looking at the `get_instance_axis_value` method, and at the check's rationale makes me think that there might be a lurking bug somewhere. But I think that's better investigated and addressed in a separate PR.

As for this PR, the descriptions you referred to are correct: a condition was removed from each of the checks' `conditions` list. The checks now are no longer skipped when the problems they are meant to catch are found.

Reply:
I still think the CHANGELOG entries are imprecise. If there's a bug in the `regular_wght_coord` condition we must find it and fix it. On the other hand, if there's no bug on the condition, then my understanding is that all scenarios in which is resulted in a SKIP continue to produce a SKIP with this new implementation.

I'd be happy to be proven wrong with a sample font file that has a "Regular" instance with a bad `wght` axis coordinate value and yet is inadequately skipped by this check instead of resulting in a **FAIL** in the original implementation (prior to the changes proposed in this PR)

Reply:
I'm just being a bit picky here because if there's really a lurking bug, then merging this PR and trying to address the problem in a followup issue/PR would potentially make it even trickier to find.

And if there's not lurking bug, then the changelog entries are imprecise.

Reply:
I've removed the changelog entries.
I'm happy to prove you wrong, but not until this PR is merged. For me it will be easier to investigate any possible problems in a separate PR.